### ISSUE TITLE
Staging Review finding: Incorrect heading level for document upload page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#103791]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Dependent Experience Team
**Product:** 21-686c and 21-674
**Feature:** Critical Form Updates

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/add-remove-form-21-686c-v2/add-spouse-evidence](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c-v2/add-spouse-evidence), the heading hierarchy doesn't quite match the structure of the content.

Currently, the headings go:
- H1: Add or remove a dependent on VA benefits
   - H2: Additional Information
      - H3: Submit supporting evidence to add your spouse
   - H2: [Accordion toggle] Supporting evidence you need to submit
      - H3: Submit your files online

Screen reader users often navigate a page by jumping through the headings, and their screen reader announces the heading level at each stop. That heading level helps them understand the structure of the page, and when the structure doesn't match the content it can be disorienting.

This also occurs on:
- [/add-remove-form-21-686c-v2/add-child-evidence](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c-v2/add-child-evidence)

## Recommended action

The way I read the page, I think the structure should be:

- H1: Add or remove a dependent on VA benefits
   - H2: Additional Information
      - H3: Submit supporting evidence to add your spouse
         - H4: [Accordion toggle] Supporting evidence you need to submit
         - H4: Submit your files online

Note that you can style these headings however it might make sense for the visual hierarchy of the page, so long as they're coded to reflect the structure of the content.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by Midge-dev** [comment]
All [VA Accordion](https://design.va.gov/components/accordion) headers are H2. It seems like there might be some support being built in for custom header levels (see storybook), but this functionality is not currently working and may need a bug report.

cc: @steele-lm @fiorella-io
